### ISSUE TITLE
[QUEST] Weekly raid quest "... Must Die!"

### ISSUE THREAD

**Issue by slimepool** [dalaranwow/dalaran-wow#4768]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Weekly raid quests "... Must Die!" from Archmage Lan'dalock can be taken, but cannot be turned in.
**Expected behaviour**: Tell us what should happen instead
Obviously, quests should be able to turn in.
**Steps to reproduce the problem**:

1. Take the quest from Archmage Lan'dalock (ID 20735), for example, "XT-002 Deconstructor Must Die!" (ID 24588)
2. Kill XT-002 Deconstructor in Ulduar
3. Go back to Archmage Lan'dalock

Wild guess on issue - creature_questender table is missing data on theese quests.

**Comment by Rushor** [comment]
Your issue posted here is a duplicate of issue: https://github.com/dalaranwow/dalaran-wow/issues/4766 (< click on this and you will be redirected to the original posted issue). 

As you can see the issue has been fixed by a developer and will be tested on the Public Test Realm in the future (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm in the future.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.
